contributing/release-schedule.md

```diff
@@ -1,84 +1,84 @@
 
-- The ["Type: Release" GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) for each release build will include a link to the specific [release page](https://github.com/ampproject/amphtml/releases) which lists the PRs in that release.
-  - The most recent "Type: Release" issue with `(Production)` in the title is the build currently in production.
-  - The "Type: Release" issue with `(Canary)` in the title tracks the current canary.
-- The [PR Use: In Canary](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Canary%22) and [PR Use: In Production](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Production%22) labels are added to PRs when they've made it into a canary/production build. There may be a delay between when the build is created and when it goes live.
+The **stable** release is built with the _production_ configuration and served to most AMP traffic. Since the **beta** release is also built from the _production_ configuration, it represents the exact build which will become **stable** the following week (ignoring the possibility of cherry-picks; see [Contributing Code](https://github.com/ampproject/amphtml/blob/master/contributing/lts-release.md)).
 
-## Cherry picks
+### Long-Term Stable (LTS)
 
-We have a well-defined process for handling requests for changes to the canary release build or to the current production release build. These changes are known as "cherry picks".
+The **lts** release channel provides a previous **stable** build for four-week intervals. Every four weeks, the current **stable** release is promoted to **lts**. This is not recommended for all publishers building experiences with AMP, but is provided so that those performing a QA cycle may do so less often. Individual pages can explicitly opt into **lts** (see [LTS Readme](https://github.com/ampproject/amphtml/blob/master/contributing/lts-release.md).
 
-### Cherry pick criteria
+One trade-off to be aware of is that, because the **lts** release is updated every four weeks, the release binaries may be as much as six weeks behind the `HEAD` of [`ampproject/amphtml`](https://github.com/ampproject/amphtml). For this reason, sites using the **lts** release channel should not use newly introduced features. See the section on [determining if your change is in a release](#Determining-if-your-change-is-in-a-release).
```

Comment:
Same as above --
Suggestion:
"Important: Publishers using the lts release channel should not use newly introduce features. Because of the four week cycle, the lts release may be as much as six weeks behind the HEAD of ampprojects/amphtml. See the section on determining if your change is in a release. "

If possible, could we be more specific about "newly" - it would be nice to give a timeline.

Reply:
So, I'm not sure what is considered new? Do we know when "new" features are stable for this?

Reply:
This is referring to how, if you use a feature that's only been merged 2 weeks ago, it's probably not in LTS yet. So if a publisher used `<amp-magic-thing>` because they see the PR was merged, that page may not work if the current LTS version hasn't caught up to the PR that introduced the `amp-magic-thing` component.

Reply:
I think I understand that. Bt I think we should explain to readers how to determine if that component/feature is ready to be used. "new" could mean a lot of different things to a lot of different people.

Reply:
Do you think the guidance in `#Determining-if-your-change-is-in-a-release`, along with the labels automatically added to PRs ie. _PR Use: In Canary_, _PR Use: In LTS_, etc. should be clear enough for publishers?

Reply:
This confuses me again, lol. So, determining if **my** change is in a release doesn't tell me if a feature is safe to be used. With that, if the PRs are labeled with when features/components are available to be used, that should be enough. I might be getting a bit nit picky here. Maybe update the last line to:

See the section on "determining if your change is in a release" to validate if a change will be ready with your chosen release cycle.

Reply:
Done :man_shrugging: